src/core/mempool.rs

```diff
@@ -78,6 +90,56 @@ use crate::util::db::table_exists;
 pub const MEMPOOL_MAX_TRANSACTION_AGE: u64 = 256;
 pub const MAXIMUM_MEMPOOL_TX_CHAINING: u64 = 25;
 
+// name of table for storing the counting bloom filter
+pub const BLOOM_COUNTER_TABLE: &'static str = "txid_bloom_counter";
+
+// bloom filter error rate
+pub const BLOOM_COUNTER_ERROR_RATE: f64 = 0.001;
+
+// expected number of txs in the bloom filter
+pub const MAX_BLOOM_COUNTER_TXS: u32 = 8192;
+
+// how far back in time (in Stacks blocks) does the bloom counter maintain tx records?
+pub const BLOOM_COUNTER_DEPTH: usize = 2;
+
+// maximum many tx tags we'll send before sending a bloom filter instead.
+// The parameter choice here is due to performance -- calculating a tag set can be slower than just
+// loading the bloom filter, even though the bloom filter is larger.
+const DEFAULT_MAX_TX_TAGS: u32 = 2048;
+
+#[derive(Debug, Clone, PartialEq, Hash, Eq)]
+pub struct TxTag(pub [u8; 8]);
+
+impl TxTag {
+    pub fn from_seed_and_txid(seed: &[u8], txid: &Txid) -> TxTag {
```

Comment:
Some function name bike shedding -- names like `from` are really meant to be conversion methods, and this is more like a constructor than a conversion, which usually should use `new`. Even so, it's clear from the function definition that it is "from" a seed and a txid, and there's no overloading happening (i.e., there's no other `from_x` methods), so we'd end up with more readable code if it were just `from` instead.

Reply:
sure, will use `from`